### PR TITLE
Remove loader.rb and "modernize" script/test.

### PR DESCRIPTION
**Pull request by envygeeks**
Fixes: #3573

### CODE REVIEW

**Comment by envygeeks** [comment]
Yeah I didn't mean "utitlize modernize".  I meant just modernize.

**Comment by parkr** [comment]
I remember adding the loader in because rake wasn't available in Ruby 2.2 (just the lib files weren't there).

File: script/test [hunk 1]
```diff
@@ -1,23 +1,19 @@
 #! /bin/bash
-#
+set -xe
```
Comment (parkr) [inline script/test]: Could we live without `x`?

**Comment by parkr** [comment]
:+1:

**Comment by parkr** [comment]
> I remember adding the loader in because rake wasn't available in Ruby 2.2 (just the lib files weren't there).

Seems to have been fixed!

**Pull request merged**